nixos/modules/services/web-servers/phpfpm.nix

```diff
@@ -37,8 +37,8 @@ in {
 
       phpPackage = mkOption {
         type = types.package;
-        default = pkgs.php;
-        defaultText = "pkgs.php";
+        default = pkgs.php.out;
```

Comment:
Instead of making the user to set e.g. `phpPackage = php56.out;`, keep this as it is, but instead use `${phpPackage.out}` everywhere where `phpPackage` was used by the module.